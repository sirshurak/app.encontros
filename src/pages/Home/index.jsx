import React from "react";
import useUsers from "../../hooks/UseUsers";
import UsersInfoSideSlider from "../../components/UsersInfoSideSlider";
import UsersInfoVertical from "../../components/UsersInfoVertical";

function Home() {
  const { users: likedUsers } = useUsers("liked");
  const { users: chatUsers } = useUsers("chat");

  return (
    <div className="home">
      <header className="app-header">
        <h1>Conversas</h1>
      </header>
      <UsersInfoSideSlider
        users={likedUsers}
        title={`Te curtiram ${likedUsers?.length ?? ""}`}
        ageTitle="anos"
      />
      <UsersInfoVertical users={chatUsers} title="Conversas" ageTitle="anos" />
    </div>
  );
}

export default Home;
