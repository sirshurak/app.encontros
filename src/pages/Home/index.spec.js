import { shallow } from 'enzyme';
import Home from './index';

describe('Home component', () => {
    it('should match shallow snapshop', () => {
        const component = shallow(<Home />)
        expect(component).toMatchSnapshot();
    });
});