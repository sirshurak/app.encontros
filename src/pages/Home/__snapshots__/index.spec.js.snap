// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match shallow snapshop 1`] = `
<div
  className="home"
>
  <header
    className="app-header"
  >
    <h1>
      Conversas
    </h1>
  </header>
  <UsersChatSideSlider
    ageTitle="anos"
    title="Te curtiram "
  />
  <UsersInfoVertical
    ageTitle="anos"
    title="Conversas"
  />
</div>
`;
